frontend/src/utils/ews-utils.ts

```diff
@@ -99,16 +103,16 @@ export const fetchEWSDataPointsByLocation = async (
   dispatch: Dispatch,
   externalId?: string,
 ): Promise<EWSSensorData[]> => {
-  const endDate = moment(date)
-    .clone()
-    .set({ hour: 23, minute: 59, second: 59 });
+  const endDate = new Date(date);
+  endDate.setHours(23, 59, 59, 999);
   // FIXME: pass start/end here? why the 24h delta?
-  const startDate = endDate.clone().subtract(1, 'days');
+  const startDate = new Date(endDate.getTime() - oneDayInMs);
   const format = 'YYYY-MM-DDTHH:mm:ss';
```

Comment:
Can we create an enum of all the valid formats we use across the codebase to avoid having these random strings?